packages/i18next/src/lib/types/context.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Context for fetchLanguage functions.
+ * This context enables implementation of per-guild, per-channel, and per-user localisation.
+ */
+export interface I18nContext {
+	guild: Record<string, unknown> | undefined;
+	channel: Record<string, unknown> | undefined;
+	author: Record<string, unknown> | undefined;
```

Comment:
Make the `I18nGuildContext`, `I18nChannelContext`, and `I18nUserContext` interfaces, all of them being empty (do not use `type`, we want to augment them, just ignore TypeScript ESLint).
```suggestion
	guild?: I18nGuildContext;
	channel?: I18nChannelContext;
	author?: I18nUserContext;
```

Why this? Because `/register-discordjs` can do:
```typescript
interface I18nGuildContext extends Guild {}
```

Which would effectively make `I18nGuildContext` extend `Guild` with all of its properties, even augmented ones.

Reply:
Do you mean to have the register override the context interfaces? Because if I export the interfaces normally from `/register-discordjs` normally, I get an error during build saying the context interfaces from before could not be found, which makes sense.